extensions/wikia/WAMPage/css/WAMPage.scss

```diff
@@ -162,6 +167,32 @@ $color-wam-button: #092344;
 					bottom: 0;
 				}
 			}
+
+			.placeholder-all {
```

Comment:
Is it possible to reuse classes that have those colors already defined?
This way we could avoid code repetition in output CSS.
I see ```.vertical-bg-all``` defined in https://github.com/Wikia/app/blob/CE-1226-wam-placeholder/extensions/wikia/WAMPage/css/WAMPage.scss#L23

What do you think?